docs/tutorials/writing-sd-card-image-etcher.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Writing an SD Card Image Using Etcher (Cross-Platform)
+subject: SD Card Image
+---
+
+1. Download and install Etcher from [their website](http://etcher.io).
+2. Launch Etcher after it has finished installing.
+
+   <div>{% include screenshot.html source="/images/etcher/home.png" caption="Etcher home screen" %}</div>
```

Comment:
I believe the issue is that it's indented in a list, but I am not really sure. When I don't include the divs, it HTML escapes the paragraphs and you end up with text that literally says `<p> foo bar </p>`.